spec/controllers/works/default_rails_actions_spec.rb

```diff
@@ -301,18 +301,106 @@ def call_with_params(params)
 
         context "with restricted works" do
           before do
-            @work2 = FactoryGirl.create(:work, posted: true, fandom_string: @fandom.name, restricted: true)
+            @work2 = create(:posted_work, fandom_string: @fandom.name, restricted: true)
             update_and_refresh_indexes('work')
           end
 
-          it "should not show restricted works to guests" do
+          it "shows restricted works to guests" do
             get :index, params: { tag_id: @fandom.name }
             expect(assigns(:works).items).to include(@work)
             expect(assigns(:works).items).not_to include(@work2)
           end
 
         end
 
+        context "when tag is a synonym" do
+          before do
+            @fandom_synonym = create(:fandom, merger: @fandom)
```

Comment:
We should use `let` to avoid leaking state between tests in instance variables. You don't need to change all the existing instance variables though.